recOrder/plugin/qtdesigner/recOrder_ui.py

```diff
@@ -262,6 +262,7 @@ def setupUi(self, Form):
         self.cb_calib_mode.setObjectName("cb_calib_mode")
         self.cb_calib_mode.addItem("")
         self.cb_calib_mode.addItem("")
+        self.cb_calib_mode.addItem("")
```

Comment:
My commit moves all changes in this file to `main_wiget.py` to allow QtCreator to be used in the future.